src/components/mission-planning/ContextMenu.vue

```diff
@@ -22,8 +22,9 @@
               color="#FFFFFF22"
               class="text-[12px] rotate-[145deg]"
               @click="handleToggleSurvey"
-            ></v-btn
```

Comment:
Those changes should be fixed-up/squased on the original commits.

Reply:
If the original code is from outside the PR, yes, if the original code that a subsequent commit is changing is from a commit created in this same PR, no, otherwise the review process would inevitably create a cascade of review commits, which goes against the clean-master/rebase workflow.